### PR TITLE
enhanced readability

### PR DESCRIPTION
**Pull request by lkwg82**

### CODE REVIEW

File: pitest/src/main/java/org/pitest/mutationtest/incremental/IncrementalAnalyser.java [hunk 1]
```diff
@@ -21,93 +13,89 @@
 import org.pitest.mutationtest.engine.MutationDetails;
 import org.pitest.util.Log;
 
+import java.util.*;
```
Comment (hcoles) [inline pitest/src/main/java/org/pitest/mutationtest/incremental/IncrementalAnalyser.java]: The coding style for pitest does not allow wildcard imports.

File: pitest/src/main/java/org/pitest/mutationtest/incremental/IncrementalAnalyser.java [hunk 2]
```diff
@@ -21,93 +13,89 @@
...
+import java.util.Map.Entry;
+import java.util.logging.Logger;
+
 public class IncrementalAnalyser implements MutationAnalyser {
 
-  private static final Logger              LOG         = Log.getLogger();
+  private final static Logger        LOG         = Log.getLogger();
 
-  private final CodeHistory                history;
```
Comment (hcoles) [inline pitest/src/main/java/org/pitest/mutationtest/incremental/IncrementalAnalyser.java]: Unless they are intended to be mutable class members should be final.

File: pitest/src/main/java/org/pitest/mutationtest/incremental/IncrementalAnalyser.java [hunk 3]
```diff
@@ -21,93 +13,89 @@
...
       }
     }
 
   }
 
-  private MutationResult analyseFromHistory(final MutationDetails each,
-      final MutationStatusTestPair mutationStatusTestPair) {
+  private MutationResult analyseFromHistory(MutationDetails each, MutationStatusTestPair mutationStatusTestPair) {
 
-    final ClassName clazz = each.getClassName();
```
Comment (hcoles) [inline pitest/src/main/java/org/pitest/mutationtest/incremental/IncrementalAnalyser.java]: In large methods local variables should be declared final if they have a wide scope

File: pitest/src/main/java/org/pitest/mutationtest/incremental/IncrementalAnalyser.java [hunk 4]
```diff
@@ -29,85 +29,77 @@
...
+    return mutationResultSet;
 
   }
 
-  private void logTotals() {
-    for (final Entry<DetectionStatus, Long> each : this.preAnalysed.entrySet()) {
-      if (each.getValue() != 0) {
-        LOG.fine("Incremental analysis set " + each.getValue()
-            + " mutations to a status of " + each.getKey());
+  private static void logTotals(Map<DetectionStatus, Long> preAnalysed) {
```
Comment (KyleRogers) [inline pitest/src/main/java/org/pitest/mutationtest/incremental/IncrementalAnalyser.java]: Whats the point here? Why do you think is a private instance method more readable than a static method with a parameter?
That is just changing object oriented style into procedural style.

Comment (lkwg82) [inline pitest/src/main/java/org/pitest/mutationtest/incremental/IncrementalAnalyser.java]: I agree, this makes it worse.

**Comment by lkwg82** [comment]
Needs anything done to merge this, except solve merge conflicts?

**Pull request closed without merging**